src/core/index.ts

```diff
@@ -337,11 +337,14 @@ export class ThirdwebSDK {
 
   /**
    *
-   * @param address - The contract address of the given Collection module.
-   * @returns The Collection Module.
+   * @param address - The contract address of the given Bundle module.
+   * @returns The Bundle Module.
```

Comment:
We should properly document deprecation so it shows up in the doc https://tsdoc.org/pages/tags/deprecated/